processing/art/fireworks/fireworks.js

```diff
@@ -0,0 +1,250 @@
+// Forked from https://github.com/andrewdcampbell/jsfireworks/blob/master/sketch.js
```

Comment:
TODO: Indicate that you have to run this via a server, like `python3 -m http.server` so it can load the image.

Reply:
https://github.com/calebmeyer/generative-art/commit/7da3b357ec2ca16aa3206016be5a0e48b0c0ba9a